source/manual/debian-packaging.html.md

```diff
@@ -153,6 +164,31 @@ Note that whilst - typically - purging all is an expensive operation
 which could take some time), in this case, the low amount, and type, of
 traffic this service receives means it's safe.
 
+#### Use a local repo in an app
+
+To make a repository available on a machine it is necessary to include a class implementing
+an `apt::source` object, e.g.:
+
+```
+class govuk_python::repo (
+  $apt_mirror_hostname = undef,
+) {
+  apt::source { 'govuk-python':
+    location     => "http://${apt_mirror_hostname}/govuk-python",
+    release      => $::lsbdistcodename,
+    architecture => $::architecture,
+    key          => 'DA1A4A13543B466853BAF164EB9B1D8886F44E2A';
+  }
+}
+```
+After this, packages can be included in the form:
+```
```

Comment:
I think there needs to be a blank line before the backticks, otherwise it forms part of the previous sentence.